server/base/src/test/java/org/apache/accumulo/server/master/balancer/HostRegexTableLoadBalancerTest.java

```diff
@@ -443,15 +443,95 @@ public void testUnassignedWithNoTServers() {
     for (TServerInstance r : removals) {
       current.remove(r);
     }
-    this.getAssignments(Collections.unmodifiableSortedMap(allTabletServers),
+    this.getAssignments(Collections.unmodifiableSortedMap(current),
```

Comment:
I think this is an issue with the existing test?  My understanding is we want to pass in a partial set of the TServers.